src/twisted/conch/test/test_transport.py

```diff
@@ -1966,6 +1948,31 @@ def test_KEX_DH_GEX_REQUEST(self, keyAlgorithm=b"ssh-rsa"):
         self.assertEqual(self.proto.g, 2)
         self.assertEqual(self.proto.p, dhPrime)
 
+    def test_KEX_DH_GEX_REQUEST_legacyFacotry(self):
```

Comment:
We shouldn't be adding new tests without type hints.

```suggestion
    def test_KEX_DH_GEX_REQUEST_legacyFacotry(self) -> None:
```